static/app/views/performance/landing/widgets/widgets/vitalWidgetMetrics.tsx

```diff
@@ -285,21 +294,27 @@ export function VitalWidgetMetrics(props: PerformanceWidgetProps) {
   );
 }
 
-function getVitalData(
-  transaction: string,
-  field: string,
-  response: MetricsApiResponse | null
-) {
+export function getVitalData({
+  transaction,
+  field,
+  response,
+}: {
+  field: string;
+  response: MetricsApiResponse | null;
+  transaction?: string;
+}) {
   const groups =
-    response?.groups.filter(group => group.by.transaction === transaction) ?? [];
+    (transaction
+      ? response?.groups.filter(group => group.by.transaction === transaction)
+      : response?.groups) ?? [];
 
   const vitalData: VitalData = {
     poor: 0,
     meh: 0,
     good: 0,
     p75: 0,
     ...groups.reduce((acc, group) => {
-      acc[group.by.measurement_rating] = group.totals[field];
+      acc[group.by.measurement_rating] = group.totals[field] ?? 0;
```

Comment:
Is there a difference on the chart if we pass 0 vs undefined?

Reply:
yes... if I am not mistaken  in case some of the  `vitalData` property is `undefined` it will cause an error in another component , but i would have to double-check it